modules/backend/lang/sk/lang.php

```diff
@@ -15,6 +16,11 @@
     ],
     'page' => [
         'untitled' => 'Bez názvu',
+        '404' => [
+            'label'     => 'Stránka nenájdená',
+            'help'      => "Hľadali sme a hľadali, ale požadovanú adresu URL jednoducho nebolo možné nájsť. Možno ste hľadali niečo iné?",
+            'back_link' => 'Späť na predchodzú stránku',
```

Comment:
@Samuell1 Would you be able to remove the extra spaces before the arrow indicators? The rest of the document seems to be using the `space arrow space` format.

Reply:
I can but this was copied from en lang.php file

Reply:
I'm fine with the aligned => format, eventually I'd like it everywhere in the lang files

Reply:
Anything needed? @LukeTowers